worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionFilesSorter.java

```diff
@@ -89,11 +96,17 @@ public class PartitionFilesSorter extends ShuffleRecoverHelper {
   private final ExecutorService fileSorterExecutors;
   private final Thread fileSorterSchedulerThread;
 
+  private final ForkJoinPool forkJoinPool =
+      new ForkJoinPool(Runtime.getRuntime().availableProcessors());
```

Comment:
that's too dangerous when deploying Celeborn on the bare metal machine, it would swallow all cores

Reply:
it should be config